src/cmd/ksh93/bltins/typeset.c

```diff
@@ -275,8 +268,8 @@ int    b_typeset(int argc,register char *argv[],Shbltin_t *context)
 					flag |= NV_HEXFLOAT;
 				}
 				else
-				        /* n=='F' Remove possible collision with NV_UNSIGN/NV_HEXFLOAT */
-				        flag &= ~NV_HEXFLOAT;
+					/* n=='F' Remove possible collision with NV_UNSIGN/NV_HEXFLOAT */
```

Comment:
I did this code. Please teach m why this is wrong?

Reply:
You used both spaces and tabs to indent. I replaced the spaces with tabs for consistency. From policy 7 in the README:
> Your changes should match the style of the code surrounding it. Indent with **tabs**, assuming an 8-space tab width.

Reply:
Did those indentations actually produce gcc warnings? If not, it's hardly a big problem and I'm not sure I would have bothered, but it doesn't hurt to fix it either.

Reply:
These indentations weren't the ones causing warnings (the indentations that caused warnings were in `nvdisc.c` and `regsubexec.c`).